README.md

```diff
@@ -1,20 +1,26 @@
 # cell-types-analysis
-A suite of scripts for analysis of cell type classification tools outputs. 
+A suite of scripts for analysis of scRNA-seq cell type classification tool outputs. These can be used both for evaluating the existing methods by running pipelines on labelled data and for analysing predicted labels for novel data sets.  
 
-## Usage 
+## Usage
+
+### Tool evaluation scenario 
```

Comment:
Could you mention the scripts by name in this passage so there is context as to how each script fits with the objectives?